fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -465,6 +506,11 @@ struct TenantManagementWorkload : TestWorkload {
 					ASSERT(entry.get().tenantGroup == tenantItr->second.tenantGroup);
 					ASSERT(entry.get().tenantState == TenantState::READY);
 
+					if (operationType != OperationType::SPECIAL_KEYS) {
```

Comment:
Why doesn't this work with special keys?

Reply:
I misunderstood how special keys worked, changed to include it